transforms/goog_module_to_es6_module.js

```diff
@@ -75,6 +77,10 @@ module.exports = (info, api, options) => {
       throw new Error('Could not transform symbol ' + symbol + '; note that destructuring is not supported');
     }
 
+    if (!currentModuleSymbol) {
+      new Error('The goog.module is required to requirs dependency');
```

Comment:
There is a typo and is this code required?
`currentModuleSymbol` is now always defined.